nhsx/nhsx-website
# Prod release

* Set `CACHEOPS_DEGRADE_ON_FAILURE` to `True`

### Testing

* List env vars for staging `dalmatian service list-environment-variables -i nhsx-website -s web -e staging`
* Set `REDIS_HOST` to something incorrect
* Restart the containers `dalmatian service restart-containers -s web -e staging -i nhsx-website`
* Keep Papertrail open and watch for errors like `RuntimeWarning: The cacheops cache is unreachable!`
* Check that the site is still running
* Set the `REDIS_HOST` back to it's previous value
* Restart the containers again
